src/components/notes/editor.jsx

```diff
@@ -198,11 +198,15 @@ const Editor = translate()(
     useEffect(() => forceSync, [docId, doc])
     // Sync on unload will probably be stopped by the browser,
     // as most async code on unload, but let's try anyway
-    const emergencySync = useCallback(function() {
-      if (doc && docId) {
-        serviceClient.sync(docId) // force a server sync
-      }
-    }, [])
+    const emergencySync = useCallback(
+      function() {
+        console.log('emergencySunc run', doc, docId)
```

Comment:
console.log should be removed

Reply:
oops. this wasn't catched by the linter

Reply:
Weird that the linter did not catch this, this might hide something bigger ?

Reply:
Yep, it seems that `console` does not trigger the linter on cozy-script v2.